easybuild/framework/easyconfig/parser.py

```diff
@@ -106,7 +109,7 @@ def process(self, filename=None):
         self._read(filename=filename)
         self._set_formatter(filename)
 
-    def check_values_types(self, cfg):
+    def check_values_types(self, cfg, auto_convert=True):
```

Comment:
no need for an `auto_convert` named argument here, use `self.auto_convert` instead?